docs/testing/sideload-office-add-ins-for-testing.md

```diff
@@ -1,57 +1,62 @@
-    > Note that in both cases, however, you can only load manifests from localhost.
+1. Open [Office on the web](https://office.live.com/) or OneDrive. Using the **Create** option, make a document in **Excel**, **OneNote**, **PowerPoint**, or **Word**. In this new document, select **Share** in the ribbon, select **Copy Link**, and copy the URL.
 
-1. In the command line starting at the root directory of your project, run the following command. Replace "{url}" with the URL of an Office document on your OneDrive or a SharePoint library to which you have permissions.
+1. In the command line starting at the root directory of your project, run the following command. Replace "{url}" with the URL that you copied.
 
     [!INCLUDE [npm start:web command syntax](../includes/start-web-sideload-instructions.md)]
 
 1. The first time you use this method to sideload an add-in on the web, you'll see a dialog asking you to enable developer mode. Select the checkbox for **Enable Developer Mode now** and select **OK**.
 
 1. You will see a second dialog box, asking if you wish to register an Office Add-in manifest from your computer. You should select **Yes**.
 
-1. Your add-in is installed. If it is an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in, the task pane should appear.
+1. Your add-in is installed. If it has an add-in command, it should appear on either the ribbon or the context menu. If it is a task pane add-in withour any add-in commands, the task pane should appear.
+
+## Sideload an add-in on the web when using Visual Studio
+
+If you're using Visual Studio to develop your add-in, pressing F5 will automatically open an Office document in *desktop* Office, create a blank document, and sideload the add-in. When you want to sideload to *Office on the web*, the process to sideload is similar to manual sideloading to the web. The only difference is that you must update the value of the **SourceURL** element, and possibly other elements, in your manifest to include the full URL where the add-in is deployed.
+
+1. In Visual Studio, show the **Properties** window by choosing **View** > **Properties Window**.
+1. In the **Solution Explorer**, select the web project. This will display properties for the project in the **Properties** window.
+1. In the Properties window, copy the **SSL URL**.
+1. In the add-in project, open the manifest XML file. Be sure you are editing the source XML. For some project types Visual Studio will open a visual view of the XML which will not work for the next step.
+1. Search and replace all instances of **~remoteAppUrl/** with the SSL URL you just copied. You will see several replacements depending on the project type, and the new URLs will appear similar to `https://localhost:44300/Home.html`.
+1. Save the XML file.
+1. Right click the web project and choose **Debug** > **Start new instance**. This will run the web project without launching Office.
```

Comment:
We should avoid using right click. Or you should provide an alternative instruction that works for other entry modes (e.g. keyboard). Linda Lu may have some suggestions.
```suggestion
1. Open the context menu of the web project (for example, by right clicking on it) then choose **Debug** > **Start new instance**. This runs the web project without launching Office.
```